### ISSUE TITLE
Windows 7 debugging information is blank.

### ISSUE THREAD

**Issue by asadsahi** [OmniSharp/omnisharp-vscode#379]
System information:

C:\Users\asahi>dotnet --info
.NET Command Line Tools (1.0.0-preview1-002702)

Product Information:
 Version:     1.0.0-preview1-002702
 Commit Sha:  6cde21225e

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64

VS code version:
![image](https://cloud.githubusercontent.com/assets/3470892/15573584/ed84f232-233f-11e6-91f5-559c06195f0e.png)

C# extension is latest:
![image](https://cloud.githubusercontent.com/assets/3470892/15573598/045fe6b0-2340-11e6-9b0e-18fe50441cec.png)

.Net core debugger installed successfully when asp.net core rc2 project is loaded. 

When debugging using visual studio code, here is what the picture looks like:

![image](https://cloud.githubusercontent.com/assets/3470892/15573582/e5ac029e-233f-11e6-9a6b-0bfda85933ab.png)

Observations:
1) Variables don't get populated
2) cannot add anything in watch
3) In command window typing any variable in context result in strage error (see at the bottom of screenshot above)
4) all debugger tooltips don't show anything
5) Breakpoint gets hit though.

In summary on windows 7 any .net core project debugging information is not available. However the same project's debugging information using all the same version of tools mentioned above, is available on windows 10.

Is this a known issue or I have missed something?

Thanks

**Comment by gregg-miskelly** [comment]
This is #258 - yup, unfortunately expression evaluation is pretty much entirely broken at the moment. We are trying to get it fixed.

**Comment by asadsahi** [comment]
Thanks @gregg-miskelly looking forward to the fix :)
